src/sgame/sg_admin.cpp

```diff
@@ -5791,7 +5791,7 @@ static void BotUsage( gentity_t *ent )
 	                                        "            bot del (<name> | all)\n"
 	                                        "            bot names (aliens | humans) <names>…\n"
 	                                        "            bot names (clear | list)\n"
-	                                        "            bot behavior (<name> | <slot#>) <behavior> [<x> <y> <z>]\n"
+	                                        "            bot behavior (<name> | <slot#>) <behavior> [<name> | <slot#> | <x> <y> <z>]\n"
```

Comment:
maybe you can specify that it's the _followed_ name and _followed_ slot number, because if I didn't know already what it was because of the context I would have a hard time guessing

Reply:
It is not really only the followed name/number. This is a general mechanism to pass arguments to .bt files, albeit restricted to a single vector or a single client number. Other .bt files might do something completly different with it.

I know, I know we should have a mechanism capable of passing more arguments. But this is the minimal interface I need.